capacity4more/modules/c4m/features_og/c4m_features_og_events/plugins/content_types/upcoming_events/upcoming_events.inc

```diff
@@ -15,7 +15,7 @@ $plugin = array(
 function c4m_features_og_events_upcoming_events_content_type_render($subtype, $conf, $args, $context) {
```

Comment:
Create 2 different renderes:

1. Upcoming event for non-logged in users.
2. Upcoming events for logged in users.